### PR TITLE
Add metric to track how many times is each key requested

### PR DESCRIPTION
**Pull request by lbaquerofierro**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -81,6 +81,10 @@ export const handleTokenRequest = async (ctx: Context, request: Request) => {
 	const signedToken = await issuer.issue(tokenRequest);
 	ctx.metrics.signedTokenTotal.inc({ env: ctx.env.ENVIRONMENT });
```
Comment (thibmeu) [inline src/index.ts:82]: could you add `key_id: keyId` here instead? This is the same metric overall, just adding one label.

Comment (lbaquerofierro) [reply src/index.ts]: Added the label + service label

**Comment by fisherdarling** [comment]
I have a concern with cardinality here. We usually try not to add prometheus labels with unbounded cardinality, due to each unique label creating a unique prometheus timeseries. I guess the kids will be short-lived ~1 week, so this might be fine.

**Comment by thibmeu** [comment]
@fisherdarling cardinality of key_id is 256 (from 0 to 255). I agree this is large.
The number of values then depends on the prometheus server retention time, speed of the rotation, and numbers of keys to retain in parallel.
If rotation is once a day and retention of two weeks, the upperbound is 14 (for 14 days).

**Comment by thibmeu** [comment]
@lbaquerofierro  I've posted a change proposal to your repo https://github.com/lbaquerofierro/privacypass-issuer/pull/1

It does not create an additional metric, and allows to set default labels.

**Review by thibmeu — CHANGES_REQUESTED** [review]

**Comment by fisherdarling** [comment]
> @fisherdarling cardinality of key_id is 256 (from 0 to 255). I agree this is large. The number of values then depends on the prometheus server retention time, speed of the rotation, and numbers of keys to retain in parallel. If rotation is once a day and retention of two weeks, the upperbound is 14 (for 14 days).

Great! Thanks for walking through the true cardinality. 14 is perfectly fine.

**Review by thibmeu — APPROVED** [review]
I proposed one additional commit https://github.com/lbaquerofierro/privacypass-issuer/pull/2 that you should be able to merge.
Once done, we can rebase and merge this branch

**Pull request merged**